### PR TITLE
peeknpoke for androidIA

### PR DESCRIPTION
**Pull request by jiankunx**
[debug and trace] add peeknpoke for androidIA
Tracked-By: https://jira01.devtools.intel.com/browse/OAM-44336
Signed-off-by: btian1 <baofeng.tian@intel.com>
Signed-off-by: jiankunx <jiankunx.liu@intel.com>

### CODE REVIEW

File: androidia_64/device.mk [hunk 1]
```diff
@@ -430,6 +430,15 @@ CRASHLOGD_MODULE_MODEM ?= true
 CRASHLOGD_USE_SD := false
 endif
```
Comment (kalyankondapally) [inline androidia_64/device.mk]: commit message is missing jira and tests. Related mixin changes ?

Comment (btian1) [inline androidia_64/device.mk]: sorry, add it now.

Comment (btian1) [inline androidia_64/device.mk]: btw, Kalyan, no worry about "should this be restricted for certain builds only ?"
it only show up in user debug build: check below for your reference:
PRODUCT_PACKAGES_DEBUG

**Comment by Kishore409** [comment]
Merged , Thanks

**Pull request merged**